docs/source/markdown/podman-machine-init.1.md

```diff
@@ -55,6 +55,17 @@ Memory (in MB).
 
 Start the virtual machine immediately after it has been initialized.
 
+#### **--volume**, **-v**=*source:target*
+
+Mounts a volume from source to target.
+
+Create a mount. If you specify /host-dir:/machine-dir, Podman mounts _host-dir_ in the host to _machine-dir_ in the Podman machine.
+In the default operating system, the root filesystem is mounted read-only, so mounts need to be created under e.g. /mnt directory.
+
+#### **--volume-driver**
```

Comment:
It would be nice to have an example of this in the Example section too.

Reply:
Will change these in the other PR